auth/src/main/java/com/firebase/ui/auth/provider/EmailProvider.java

```diff
@@ -1,48 +0,0 @@
-package com.firebase.ui.auth.provider;
-
-import android.app.Activity;
-import android.content.Context;
-import android.content.Intent;
-import android.support.annotation.LayoutRes;
-
-import com.firebase.ui.auth.R;
-import com.firebase.ui.auth.data.model.FlowParameters;
-import com.firebase.ui.auth.ui.email.EmailActivity;
-
-public class EmailProvider implements Provider {
```

Comment:
We are deleting a formerly `public` class.  Should this have always been annotated with `@RestrictTo` or this is a real breaking change?

Reply:
Same comment for all providers.

Reply:
This was from before we used the restrict annotations, but it's always been considered private API AFAIK:
https://github.com/firebase/FirebaseUI-Android/blob/a381e13957d9e9ea71da51a6ab03cd5f7e782d37/auth/src/main/java/com/firebase/ui/auth/provider/package-info.java

Reply:
SGTM  let's just mark them all as restricted now